tests/test_samples.py

```diff
@@ -1179,7 +1179,7 @@ def test_samples_dot_plot():
         axes = samples.plot.fastkde_2d('x0', 'x1')
         assert len(axes.collections) == 5
         plt.close("all")
-        axes = samples.plot.fastkde_1d()
+        axes = samples.x0.plot.fastkde_1d()
```

Comment:
On the master branch, the original line plots the fastkde of `x0`; I don't see why this should happen. On this branch, the original line plots the fastkde for all columns, including logL, logL_birth and nlive. I think the most appropriate behaviour would be to plot `x0`-`x4`?

For the moment, as the other parts of `test_samples_dot_plot()` only do 1d plots for a single column, I have altered this line to match.

Reply:
I agree with testing it this way. Plotting all columns is the correct (albeit useless in this case) behaviour. The correct way to use this would be `samples[['x0','x1','x2','x3']].plot.fastkde_1d()`, which would only make sense for a nested sampling run if the parameters had the same dimensions (e.g. for locations of a galaxy, or angles in a protein, but not for cosmological parameters).